prost-build/src/fixtures/field_attributes/_expected_field_attributes.rs

```diff
@@ -2,33 +2,32 @@
 #[allow(clippy::derive_partial_eq_without_eq)]
 #[derive(Clone, PartialEq, ::prost::Message)]
 pub struct Container {
-    #[prost(oneof="container::Data", tags="1, 2")]
+    #[prost(oneof = "container::Data", tags = "1, 2")]
     pub data: ::core::option::Option<container::Data>,
 }
 /// Nested message and enum types in `Container`.
 pub mod container {
     #[allow(clippy::derive_partial_eq_without_eq)]
-#[derive(Clone, PartialEq, ::prost::Oneof)]
+    #[derive(Clone, PartialEq, ::prost::Oneof)]
     pub enum Data {
-        #[prost(message, tag="1")]
-        Foo(::prost::alloc::boxed::Box<super::Foo>),
```

Comment:
I believe the generated code should always use a fully qualified type path, as they could otherwise clash with a user-generated type.